HnTool/modules/ssh.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, options):
         self.short_name="ssh"
         self.long_name="Checks security problems on sshd config file"
         self.type="config"
-        self.required_files = ['/etc/ssh/sshd_config', '/etc/sshd_config']
```

Comment:
The correct approach on hntool is to ask the config file's full path, but I don't know how to do that yet!